docs/reference/api/api-httpmethods.md

```diff
@@ -0,0 +1,81 @@
+The following links all redirect here:
+
+- `api.get()` - HTTP `GET`
+- `api.post()` - HTTP `POST`
+- `api.put()` - HTTP `PUT`
+- `api.delete()` - HTTP `DELETE`
+- `api.patch()` - HTTP `PATCH`
```

Comment:
Docs looks great, just this note is a bit confusing

Reply:
I'll create another change later to fix this using a template page or something similar